Docs/source/dataanalysis/inputs.2d

```diff
@@ -34,9 +34,7 @@ warpx.cfl = .99
 warpx.do_moving_window = 1
 warpx.moving_window_dir = z
 warpx.moving_window_v = 1. # in units of the speed of light
-interpolation.nox = 3
-interpolation.noy = 3
-interpolation.noz = 3
+interpolation.shape_factors_order = 3

```

Comment:
Discussed and suggested in the the PR description:
Since this is also used for current deposition, interpolation is a bit confusing already. Moving it to `warpx.` could be a good idea.

Note: in general one could also set this per species (or overwrite a default per species). Could be for the future if we care. Adding this to the dev meeting just for curiosity.